node-repository/src/test/java/com/yahoo/vespa/hosted/provision/NodeRepositoryTest.java

```diff
@@ -71,17 +72,18 @@ public void featureToggleDynamicAllocationTest() {
     @Test
     public void only_allow_to_delete_dirty_nodes_when_dynamic_allocation_feature_enabled() {
         NodeRepositoryTester tester = new NodeRepositoryTester();
-        try {
-            tester.addNode("id1", "host1", "default", NodeType.host);
-            tester.addNode("id2", "host2", "docker", NodeType.tenant);
-            tester.nodeRepository().setDirty("host2");
-
-            assertFalse(tester.nodeRepository().remove("host2"));
+        tester.addNode("id1", "host1", "default", NodeType.host);
+        tester.addNode("id2", "host2", "docker", NodeType.tenant);
+        tester.nodeRepository().setDirty("host2");
 
-            tester.curator().set(Path.fromString("/provision/v1/dynamicDockerAllocation"), new byte[0]);
-            assertTrue(tester.nodeRepository().remove("host2"));
-        } finally {
-            tester.curator().delete(Path.fromString("/provision/v1/dynamicDockerAllocation"));
```

Comment:
Was the delete in finally not needed?

Reply:
The other tests don't use it, additionally this is the last thing that happens in the test, and a new `Curator` is created for next test as part of `@Before`, the `MockCurator` uses memory FS, so delete shouldn't be necessary.

Reply:
OK, good